chart/templates/worker.yaml

```diff
@@ -34,36 +33,27 @@ spec:
             requests:
```

Comment:
I cannot place a comment on the relevant line, but above this comment you still have

```
image: docker.io/researchableuser/sdv-svc-questionnaires:{{ .Values.tag }}
```

hardcoded to sdv

Reply:
Yes, that's because that's where the image is pushed to in Dockerhub in the Gitlab mirror of the GitHub repo. Strictly not correct, but we can leave it for now I think.